src/breakpoints/body.tsx

```diff
@@ -94,10 +94,11 @@ const BreakpointComponent = ({
     <div
       className={`breakpoint`}
       onClick={() => model.clicked.emit(breakpoint)}
+      title={breakpoint.source.path}
     >
-      <span>
-        {breakpoint.source.path} : {breakpoint.line}
-      </span>
+      <span className={'marker'}>●</span>
```

Comment:
Are the class names `marker` and `source` intentionally not prefixed?

Reply:
For now yes, as it follows the existing style (we should also change the `breakpoint` class from the wrapper `<div>`).

We will have to do a proper pass anyway with #139, but happy to make the here too.